src/components/BodyNavBar.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+function BodyNavBar() {
```

Comment:
Lets rename this  component to something more descriptive like FeedSelector.

Reply:
Done!